airflow/www/views.py

```diff
@@ -336,11 +336,19 @@ def get_int_arg(value, default=0):
         state_color_mapping = State.state_color.copy()
         state_color_mapping["null"] = state_color_mapping.pop(None)
 
+        page_title = "Airflow - DAGs"
+        site_title = conf.get(section="webserver", key="site_title", fallback="DAGs")
+        if site_title != "DAGs":
+            site_title = site_title.replace("\"", "").replace("'", "")
```

Comment:
This shouldn't be needed.

Reply:
Hi @ashb thanks for the feedback here. Remove lines 339 - 342 or something in particular?

Reply:
Just this line where we remove quotes

Reply:
I tested it yesterday before I went to sleep. It is not necessary.
<img width="542" alt="Screenshot 2020-08-06 at 00 54 02" src="https://user-images.githubusercontent.com/12058428/89546022-5f62ca00-d804-11ea-90ea-6357694d04ab.png">
<img width="574" alt="Screenshot 2020-08-06 at 00 54 36" src="https://user-images.githubusercontent.com/12058428/89546028-612c8d80-d804-11ea-99d6-a15f5a9f67a4.png">

Reply:
Thanks both, done.